composer.json

```diff
@@ -37,7 +37,8 @@
         "drupal/sitestudio_config_management": "dev-develop",
         "drupal/sitestudio_gin": "^1.0",
         "drush/drush": "^10 || ^11 || ^12",
-        "mnsami/composer-custom-directory-installer": "^2.0"
+        "mnsami/composer-custom-directory-installer": "^2.0",
+        "nnnick/chartjs": "^4.4"
```

Comment:
Can we remove this from root composer.json as it's already available in Acquia CMS COmmon module as dependency?

Reply:
Let's create a separate PR for this.